package.json

```diff
@@ -124,7 +124,8 @@
     "vite-plugin-json5": "1.1.2",
     "vite-plugin-node-polyfills": "0.22.0",
     "vite-plugin-plain-text": "1.4.2",
-    "vitest": "2.0.5"
+    "vitest": "2.0.5",
+    "fuels": "workspace:*"
```

Comment:
I had to add this so that I could run [this command in the test](https://github.com/FuelLabs/fuels-ts/pull/3528/files#diff-37ed97cd9ea1a119d0e728d36d7c5bd32d1983a202bd48688124c682515ae7c2R93). This is because the `spawn` is running in the root of the repo and there is no `fuels` dependency there that it can run. Upon adding it, the `spawn` runs as expected.